core/services/ocrbootstrap/delegate.go

```diff
@@ -23,7 +21,7 @@ type Delegate struct {
 	peerWrapper   *ocrcommon.SingletonPeerWrapper
 	cfg           validate.Config
 	lggr          logger.Logger
-	relayer       types.RelayerCtx
+	relayers      map[types.Network]types.Relayer
```

Comment:
Or with the rename suggestion:

```suggestion
	relayers      relay.Services
```

Or even:


```suggestion
	relayers      relay.Plugins
```

As we could have OCR2 plugins (different jobs), and Relay plugins (different networks).

Reply:
I think types.Relayers is actually less readable tbh, then first time readers just have to go and look up the type def to understand what the backing data structure is

Reply:
Agreed, but just the first time :)
Definitely agree that too much aliasing is no good, but it's still a tradeoff in readability.

`relay.Plugins` and `relay.Plugins[relay.EVM]` shouldn't pose too much effort for the reader IMHO, and the API branding is nice/r.

Reply:
oh actually relay.Plugins would be very confusing since types.Relayer is actually a general object which can build both a simple ConfigProvider for bootstrap jobs as well as real OCR2 plugin. We could add types.Relayers but it just doesn't seem to add much, the map is clear already